bin/run_e2e_tests

```diff
@@ -29,16 +32,16 @@ setup () {
 
 cleanup () {
     echo "Cleaning up"
-    docker-compose -p $PROJECT_NAME down
+    ${dockerComposeCmd} down
 }
 
 
 setup
 
-result=`curl localhost:4050/connection`
-if [ "$result" != '{"status":"NotConnected"}' ]
+go test -v ./e2e/... -args --tequila.host=localhost --tequila.port=4052
+if [ ! $? -eq 0 ]
 then
-    print_error "Unexpected status response: $result"
+    print_error "Tests failed"
```

Comment:
Why not cleaning containers if tests succeeds?

Reply:
It is being cleanup, look where `cleanup` function is invoked.

Reply:
it is:
print_success "Tests passed"
cleanup